packages/flame/lib/src/events/tap_config.dart

```diff
@@ -0,0 +1,27 @@
+import 'dart:math';
+
+/// [TapConfig] is used to expose specific configurations
+/// related to the tap dispatcher.
+class TapConfig {
```

Comment:
```suggestion
final class TapConfig {
```
Realized this class should probably be final.